app/views/layouts/application.html.erb

```diff
@@ -24,7 +24,7 @@
       </div>
       <div class="col-md-4 col-xs-4 text-right">
         <%= yield(:title_button) if content_for?(:title_button) %>
-         <%= link_to 'Log Out', destroy_user_session_url, class: 'btn btn-default vcenter', style: 'margin-top: 28px', method: :delete if user_signed_in? %>
+         <%= link_to 'Log Out', destroy_user_session_path, class: 'btn btn-default vcenter', style: 'margin-top: 28px', method: :delete if user_signed_in? %>
```

Comment:
@jbeast This now uses a relative path instead of the full URL, should we be doing this for all links?